pages/report/index.tsx

```diff
@@ -1,13 +1,20 @@
-import React from "react"
+import React, { useContext } from "react"
 import type { NextPage } from "next"
 import { Heading } from "@chakra-ui/layout"
+import { useTimeIntervalTimesheetEntries } from "@/lib/hooks/useTimesheetEntries"
+import { UserContext } from "@/lib/contexts/FirebaseAuthContext"
 
-const Report: NextPage = () => (
-    <div>
-        <Heading fontWeight="black" margin="1rem 0rem">
-            Reports
-        </Heading>
-    </div>
-)
+const Report: NextPage = () => {
+    const { user } = useContext(UserContext)
+    const reportsRespose = useTimeIntervalTimesheetEntries(user)
```

Comment:
We could get rid of the linter error here by replacing the current return value with
```typescript
    return (
        <div>
            {reportsRespose.isSuccess && <Heading fontWeight="black" margin="1rem 0rem">
                Reports
            </Heading>}
        </div>
    )
```